pkg/cloudprovider/aws/aws.go

```diff
@@ -1710,6 +1733,31 @@ func (s *AWSCloud) ensureSecurityGroup(name string, description string, vpcID st
 	return groupID, nil
 }
 
+// createTags calls EC2 CreateTags, but adds retry-on-failure logic
+// We retry mainly because if we create an object, we cannot tag it until it is "fully created" (eventual consistency)
+// The error code varies though (depending on what we are tagging), so we simply retry on all errors
+func (s *AWSCloud) createTags(request *ec2.CreateTagsInput) (*ec2.CreateTagsOutput, error) {
+	attempt := 0
```

Comment:
Optional (potentially) cleaner approach to this style of thing:

```go
tick := time.Tick(1 * time.Second)
for {
   select {
   case <- tick:
     ...
   case <- time.After(60 * time.Second):
   ...
}
```

You can do this if you want, definitely not required.